adapters/jsdom-react/src/spec.tsx

```diff
@@ -30,35 +30,67 @@ describe('react base driver - test suite', () => {
 });
 
 describe('react base driver specific tests', () => {
+	const spy = sinon.spy;
 
-	it('sends event data properly on simulated events when element is attached to body', async () => {
-		const cleanJsdom = require('jsdom-global')();
-		const s = spy();
-		const elem = document.createElement('div');
-		const btn = <button onClick={s}>bob</button>;
-		ReactDOM.render(btn, elem);
+	describe('click', () => {
+		it('sends event data properly on simulated events when element is attached to body', async () => {
+			const cleanJsdom = require('jsdom-global')();
+			const s = spy();
+			const elem = document.createElement('div');
+			const btn = <button onClick={s}>bob</button>;
+			ReactDOM.render(btn, elem);
 
-		const driver = jsdomReactUniDriver(elem);
+			const driver = jsdomReactUniDriver(elem);
 
-		await driver.$('button').click();
-		cleanJsdom();
+			await driver.$('button').click();
+			cleanJsdom();
 
-		assert.equal(s.lastCall.args[0].target.tagName, 'BUTTON');
-	});
+			assert.equal(s.lastCall.args[0].target.tagName, 'BUTTON');
+		});
 
-	it('sends event data properly on simulated events when element is attached to body', async () => {
-		const cleanJsdom = require('jsdom-global')();
-		const s = spy();
-		const elem = document.createElement('div');
-		document.body.appendChild(elem);
-		const btn = <button onClick={s}>bob</button>;
-		ReactDOM.render(btn, elem);
+		it('sends event data properly on simulated events when element is attached to body', async () => {
+			const cleanJsdom = require('jsdom-global')();
+			const s = spy();
+			const elem = document.createElement('div');
+			document.body.appendChild(elem);
+			const btn = <button onClick={s}>bob</button>;
```

Comment:
I think all this rendering part would be better hidden behind some `renderAndMount` function. We can reveal one from the `testKit`